spec/features/saml/loa3_sso_spec.rb

```diff
@@ -4,7 +4,7 @@
   include SamlAuthHelper
   include IdvHelper
 
-  def perform_id_verification_with_usps_without_confirming_code_then_sign_out(user)
+  def perform_id_verification_with_usps_without_confirming_code(user)
     saml_authn_request = auth_request.create(loa3_with_bundle_saml_settings)
     visit saml_authn_request
     sign_in_live_with_2fa(user)
```

Comment:
We'll also need `allow(FeatureManagement).to receive(:prefill_otp_codes?).and_return(true)` if the test doesn't already use that.